js/src/View/AppView.js

```diff
@@ -0,0 +1,87 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var UserModel = require('../Model/UserModel.js');
+var appViewTemplate = require('../templates/container.hbs');
+var TodoListView = require('./TodoListView.js');
+var TodoListCollection = require('../Collection/TodoListCollection.js');
+var TodoListModel = require('../Model/TodoListModel.js');
+var todoAppTemplate = require('../templates/todoapp.hbs');
+
+var AppView = Backbone.View.extend({
+    el: 'body',
+    events: {
```

Comment:
Sometimes white space does make things a little easier to read, so let's add a new line between `el` and `events`